source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -0,0 +1,250 @@
+===================================
+Document Data Format: Extended JSON
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+Overview
+--------
+
+JSON is a data format that encodes the values of objects, arrays, numbers, and 
+so on. The **Extended JSON** format adds special syntax to JSON objects to 
```

Comment:
I wouldn't call is special syntax.  Perhaps:

   The extended json format defines a reserved set of $-prefixed keys to represent....

Reply:
Addressed in f154afda39c6d5d83d850a77618fb0a8673b4eb5